src/components/TodoInfo/TodoInfo.scss

```diff
@@ -0,0 +1,20 @@
+.TodoInfo {
+  margin-bottom: 5px;
+  padding: 5px;
+  max-height: max-content;
+  font-size: 25px;
```

Comment:
Consider using a more appropriate value for `max-height`. `max-content` may cause layout issues on some browsers. 🌐